src/components/Supplier/AppSupplierData/AddSupplierData.tsx

```diff
@@ -1,17 +1,203 @@
-import { Flex, Box } from '@chakra-ui/react';
-import AddSupplierDataName from './AddSupplierDataName';
-import AddSupplierDataAddress from './AddSupplierDataAddress';
+import {
+  Flex,
+  Box,
+  Container,
+  Grid,
+  Input,
+  Textarea,
+  NumberInput,
+  NumberInputField,
+  VStack,
+  StackDivider,
+  Wrap,
+  WrapItem,
+} from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
+import StyledSpan from 'styled/Supplier/AppSupplierData/StyledSpan';
+import StyledWrapItem from 'styled/Supplier/AppSupplierData/StyledWrapItem';
+import StyledGrid from 'styled/Supplier/AppSupplierData/StyledGrid';
+import StyledVStack from 'styled/Supplier/AppSupplierData/StyledVStack';
 
-const AddSupplierData = () => (
-  <Flex>
-    <Box w="80em" h="26em">
-      <AddSupplierDataName />
-    </Box>
-    <Box w="6em" h="26em" />
-    <Box w="100em" h="26em">
-      <AddSupplierDataAddress />
-    </Box>
-  </Flex>
-);
+const AddSupplierData = () => {
+  const { register, handleSubmit, errors } = useForm();
+
+  const onSubmit = (data: Object) => {
+    console.log(data);
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <VStack divider={<StackDivider borderColor="gray.200" />} spacing={3} align="stretch">
+        <Wrap w="100%" h="30em">
+          <StyledWrapItem h="26em">
+            <StyledGrid templateColumns="repeat(2, 1fr)" gap={2}>
+              <Container px="2em" py="0.2em" fontWeight="Bold">
+                Nazwa
+              </Container>
+              <Box>
+                <Input
+                  border="1px"
+                  borderRadius="2em"
+                  borderColor="gray.300"
+                  name="name"
+                  ref={register({ required: 'Brak nazwy' })}
```

Comment:
Whenever you are using such things as `ref` prop, from `useForm()` hook, upgrade it to v7 API.
Do the same with other deprecated parts of react-hook-form API as I mentioned above.